control-plane/csi-driver/src/bin/node/main_.rs

```diff
@@ -93,10 +97,26 @@ impl AsyncWrite for UnixStream {
 const GRPC_PORT: u16 = 50051;
 
 pub(super) async fn main() -> anyhow::Result<()> {
+    // Parse command line arguments.
     let matches = clap::Command::new(utils::package_description!())
         .about("k8s sidecar for IoEngine implementing CSI among others")
         .version(utils::version_info_str!())
         .subcommand_negates_reqs(true)
+        .arg(
+            Arg::new("rest-endpoint")
+                .long("rest-endpoint")
+                .env("ENDPOINT")
+                .default_value("http://ksnode-1:30011")
+                .help("A URL endpoint to the control plane's rest endpoint")
+                .required(true),
+        )
+        .arg(
+            Arg::new("instance-endpoint")
```

Comment:
what's this?

Reply:
Ip of the pod, or node for a daemonset.

Reply:
But we want to send a socket via the registration right?
So maybe we can use similar to other services:
> - "--grpc-endpoint=$(MY_POD_IP):50053"